pkg/hubble/filters/cel_expression.go

```diff
@@ -0,0 +1,126 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Hubble
+
+package filters
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+
+	flowpb "github.com/cilium/cilium/api/v1/flow"
+	v1 "github.com/cilium/cilium/pkg/hubble/api/v1"
+	"github.com/cilium/cilium/pkg/logging"
+	"github.com/cilium/cilium/pkg/logging/logfields"
+
+	"github.com/google/cel-go/cel"
+)
+
+var (
+	celFilterLogger = logging.DefaultLogger.WithField(logfields.LogSubsys, "cel-filter")
+	// unfortunately, "flow" conflicts with the protobuf package name "flow", so
+	// we have to use something else.
+	// TODO: what should this be?
+	flowVariableName = "_flow"
```

Comment:
That's a good question. Some ideas:

- Shorten it to `f`
- Lengthen it to `hubble_flow`
- Misspell it as `flaw` :laughing:

Reply:
I think I prefer `_flow` to most of those to be honest.

Reply:
Yeah, I'm not convinced either, was just throwing some ideas around.

Reply:
I think we can start with `_flow` and we can change it later since it's still experimental, what do you think?